### ISSUE TITLE
Is it possible to control bebop 2?

### ISSUE THREAD

**Issue by jayden118** [JackrabbitMobile/WatchToDrone#2]
Hi, 

I have compiled the code successfully but I think the code is only working for BLE drones. 
May I know how to make it works in Bebop 2 instead? 

Thank you very much.

**Comment by mwanninger** [comment]
@jayden118 Correct, this will only work for BLE drones, and was only tested on the Airborne Cargo drone.

Likely needs an update was well as I believe the Parrot SDK has been updated since then.
